### ISSUE TITLE
[whitelist] Galactic Civilizations II: Ultimate Edition (202200)

### ISSUE THREAD

**Issue by JR1994** [ValveSoftware/Proton#229]
I am requesting that Galactic Civilizations II be added to the whitelist. When you first launch the game you get a few warnings about screen resolutions. Hit OK through them the game will launch in 1024X768. Go to options and set the video to 1280X1024 and restart the game. You will still get those same warnings but the game will launch and as far I have seen play with no issues.

**Comment by Gfurst** [comment]
Nice, I'm going to this it out shortly as well.

**Comment by andrebadaia** [comment]
Confirmed.

Distro: Linux Mint 19 Tara
Processor: AMD Ryzen 7 1700
GPU: GTX 1070, Nvidia proprietary driver 396.54
RAM: 32GB
Kernel (custom) - 4.17.14 (optimized for Ryzen)
Resolution: 1920x1080
Proton 3.7-3

**Comment by Amosar** [comment]
I have open a thread on the game forum about the warning messages. 
I don't know if it will do anything about that but at least I have tried ^^
https://steamcommunity.com/app/202200/discussions/0/3338745825905752646/

**Comment by Gfurst** [comment]
Yeah, besides the warning messages the game works fine, but it seems fonts are sort of screwed up, with some minor glitches, might have something to do with fonts wine uses, so there may be something to improve here.

**Comment by soredake** [comment]
Game works fine with vanilla proton 5.0-2.
